Automattic/jetpack
# Admin Page: Remove unused dir _inc/root-child

This was left there from a file copy, committed by mistake and is not used.

#### Changes proposed in this Pull Request:

* Removes the dir `_inc/root-child`.

#### Testing instructions:

* check this branch
* run `yarn build` an expect the build to succeed.
